src/utils/errorTracking/index.js

```diff
@@ -1,5 +1,5 @@
 import Honeybadger from 'honeybadger-js'
-import {honeybadgerKey} from './keys'
+import {honeybadgerKey} from 'utils/keys'
```

Comment:
No idea why this was working before, it wasn't referencing an existing file; fixed